requirements.txt

```diff
@@ -45,7 +45,8 @@ PyYAML
 requests
 riggerlib>=2.0
 scp
-selenium
+# since 3.0 uses marionette by default
+selenium==2.53.1
```

Comment:
@RonnyPfannschmidt what if we just did selenium<3.0.0 ? They might release dome bugfix for 2.x version ...